src/main/webapp/app/view/project/screenplay/filter/project-screenplay-filter.component.html

```diff
@@ -38,5 +38,57 @@
                 </dx-slider>
             </div>
         </li>
+
+        <li class="nav-item col-lg-3 col-md-4 col-sm-6 col-xs-12">
+            <div class="form-group">
+                <label class="form-control-label"
+                       jhiTranslate="eventPlannerApp.project.screenplay.filter.from"
+                       for="field_from">From</label>
+                <dx-date-box
+                    class="dx-form-control"
+                    name="from"
+                    id="field_from"
+                    type="date"
+                    [value]="from"
+                    (valueChange)="dateValueChanged('from', $event)"
+                    [openOnFieldClick]="true"
+                    [displayFormat]="dateFormat"
+                    [min]="project!.startTime!.toDate()"
+                    [max]="project!.endTime!.toDate()"
+                    placeholder="{{dateFormat}}">
+                </dx-date-box>
+            </div>
+        </li>
+
+        <li class="nav-item col-lg-3 col-md-4 col-sm-6 col-xs-12">
+            <div class="form-group">
+                <label class="form-control-label"
+                       jhiTranslate="eventPlannerApp.project.screenplay.filter.to"
+                       for="field_to">To</label>
+                <dx-date-box
+                    class="dx-form-control"
+                    name="to"
+                    id="field_to"
+                    type="date"
+                    [value]="to"
+                    (valueChange)="dateValueChanged('to', $event)"
+                    [openOnFieldClick]="true"
+                    [displayFormat]="dateFormat"
+                    [min]="minToDate"
+                    [max]="project!.endTime!.toDate()"
+                    placeholder="{{dateFormat}}">
+                </dx-date-box>
+            </div>
+        </li>
+
+        <li class="nav-item col-lg-3 col-md-4 col-sm-6 col-xs-12">
+            <div class="btn-group">
+                <button type="button"
+                        class="btn btn-negative btn-gray"
+                        (click)="resetFilter()"
+                        jhiTranslate="eventPlannerApp.project.screenplay.filter.reset">Reset
+                </button>
+            </div>
+        </li>
```

Comment:
Evtl würde ich es in das gleiche `li` wie der Expand-Button tun.
Denke du möchtest noch andere Filters entwickeln. Dann kann sich die ganze Anordnung noch paar mal wechseln. Somit kannst du es für den Moment so sein lassen.

Reply:
wäre eventuell was und dann eine `btn-group` machen. ich schau mal noch wies aussieht.